src/convertToObject.js

```diff
@@ -12,7 +12,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+  const strArr = sourceString.trim().replace(/\s+/g, ' ').split(';');
+
+  strArr.forEach(string => {
```

Comment:
you could also try to do it with reduce, where you pass an empty object as an initial value of the accumulator.